### ISSUE TITLE
Sensitive Touch should also make you far more likely to crash through the windshield on impact

### ISSUE THREAD

**Issue by iCeCoCaCoLa64** [gta-chaos-mod/ChaosModV#456]
Currently, the Sensitive Touch effect only affects the player while on foot, but I think this should be extended to affect vehicles as well. If you're in a car, you should fly through the windshield at even the lightest impact. If you're on a bike, you should fall off just as easily.  
Things like planes and tanks would remain unaffected, but maybe that's for the best lol

**Comment by anto4440** [comment]
Alright, im not certain about his, but i believe you need to use something else than effect suggestion (which is only for new effects, improving an effect is... something else, i dont know where). But i could be very wrong.

Anyway, that is actually a great idea! It would make sense and be fun. Good job.

**Comment by iCeCoCaCoLa64** [comment]
> Alright, im not certain about his, but i believe you need to use something else than effect suggestion (which is only for new effects, improving an effect is... something else, i dont know where). But i could be very wrong.

Yeah, sorry, I juggled between Feature Request and Effect Suggestion, but I figured since it's a suggestion _about_ an effect, the latter would fit best.

**Comment by JustTunerYT** [comment]
You could rename this issue into [Effect Improvement] or [Effect Improvement Suggestion], so it would made much more sense
